spec/lib/hyrax/hirmeos/client_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+require 'hyrax/hirmeos/client'
+require 'rails_helper'
+
+RSpec.describe Hyrax::Hirmeos::Client do
+  WebMock.allow_net_connect!
+  subject(:client) { described_class.new("archivist1@example.com", "password", "https://metrics-api.operas-eu.org/events", "https://tokens.ubiquity.press/", "https://translator.ubiquity.press") }
+
+  before do
+    WebMock.allow_net_connect!
```

Comment:
I think you may be able to avoid all of the allow/disallow flipping on webmock if the webmock setup in rails_helper/spec_helper allows fedora and solr...something like this: https://github.com/avalonmediasystem/avalon/blob/master/spec/rails_helper.rb#L107

Reply:
That is really useful, thank you :)